tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -796,36 +883,21 @@ def test_kicks_off_slack_async_job(
         self, mock_uuid4, mock_find_channel_id_for_alert_rule, mock_get_channel_id
     ):
         mock_uuid4.return_value = self.get_mock_uuid()
-        self.integration, _ = self.create_provider_integration_for(
-            self.organization,
-            self.user,
-            provider="slack",
-            name="Team A",
-            external_id="TXXXXXXX1",
-            metadata={"access_token": "xoxp-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxxxx"},
-        )
         valid_alert_rule = {
             **self.alert_rule_dict,
             "triggers": [
                 {
                     "label": "critical",
                     "alertThreshold": 200,
-                    "actions": [
-                        {
-                            "type": "slack",
-                            "targetIdentifier": "my-channel",
-                            "targetType": "specific",
-                            "integration": self.integration.id,
-                        }
-                    ],
+                    "actions": self.slack_action,
                 },
             ],
         }
         with self.feature(["organizations:incidents"]):
             resp = self.get_success_response(
                 self.organization.slug, status_code=202, **valid_alert_rule
             )
-        resp.data["uuid"] = "abc123"
+        assert resp.data["uuid"] == "abc123"
```

Comment:
This was supposed to be an assertion but [2023 me](https://github.com/getsentry/sentry/commit/697e6385d69ed3004df5c31ece1a41cc30c784b0) made a mistake